DataScience/GpxWavLabeler.py

```diff
@@ -605,115 +814,110 @@ def slice_source_audio_by_cluster(
         "-D",
         "--data-home",
         default=str(Path("~").expanduser() / "Data" / "AISonobuoy"),
-        help="the directory containing GPX and WAV files",
+        help="the directory containing all GPX, WAV, and JSON files",
     )
     parser.add_argument(
         "-c",
-        "--json-filename",
+        "--collection-filename",
```

Comment:
@ralatsdc where does this file come from?

Reply:
An example file exists in the repository.

Reply:
I think I miss-understood this comment. This is the collection, not the sampling file. I list the collection file from the basin collection, since it is all we have right now. Are you suggesting that I commit the collection file? Might be a good idea.

Reply:
Move both files into the repo in a subdirectory "data" of the "Datascience" directory. I think this addresses the comment, and provides a better structure.